### DISCUSSION TITLE
AutoMarkFolderRead v1.01

### DISCUSSION THREAD

**Discussion by thirdinsight** [thirdinsight/MarkFolderRead#2] [General]
Main change is an add-on options panel where the user can choose common folders (like spam, drafts etc) where messages will be marked automatically as read.

<hr /><em>This discussion was created from the release <a href='https://github.com/thirdinsight/AutoMarkFolderRead/releases/tag/v1.01'>AutoMarkFolderRead v1.01</a>.</em>

**Comment by davidrobertevans** [comment]
Hi! This looks like a brilliantly useful add-on. Is there any way to add other folders? I have a big tree of them, and use Thunderbird's filtering feature to send a lot of incoming emails to various folders. It would just grand if I could mark these unread all the time (i.e. if I have a friend called Joe Bloggs who sends me tiresome circulars I need to keep, I can just have them end up in the Joe Bloggs folder without being bothered by seeing they have come). But just being able to add any folders in local folders (i.e. not just junk, drafts, etc) would be very useful!

**Reply by thirdinsight** [reply]
Hello,
thank you for your message!

Others gave bad ratings to the plugin because this feature does not exist, though in my opinion it has nothing to do with the plugin as it currently exists.

I thought from the beginning about implementing it (although I don't need it myself), but firstly it would cost me a lot of work and energy, which I don't have that much anymore. And secondly, I couldn't think of an acceptable solution to implement it. Later I thought that an addition to the context menu in the tree view with a "keep marked as read" selection element wouldn't be bad.

Only a short time after I published my plugin, another plugin was released which is based on the desired feature. (personally, I'm not quite convinced by its solution.) If you didn't know about this plugin yet, you can take a look at it.

Maybe I will start, but I can't promise anything.

Kind Regards
